spec/features/idv/steps/in_person/ssn_spec.rb

```diff
@@ -111,25 +110,27 @@
       # location page
       complete_location_step(user)
       # state ID page
-      fill_out_state_id_form_ok(same_address_as_id: false, capture_secondary_id_enabled: true)
+      fill_out_state_id_form_ok(same_address_as_id: false)
       click_idv_continue
-      fill_out_address_form_ok(double_address_verification: true, same_address_as_id: false)
+      fill_out_address_form_ok(same_address_as_id: false)
```

Comment:
Not stoping - maybe in the future- I feel like only `fill_out_state_id_form_ok` should take in same_address_as_id arg and fill out address form and we should not have to fill out address. Only when diff we do- but this is the current pattern for original flow- no need to change- just a comment for later.

Reply:
yes I think we should explore that option in [LG-11054](https://cm-jira.usa.gov/browse/LG-11054)